<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSeriesAdd</class>
 <widget class="QDialog" name="TimeSeriesAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Line</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelStation">
       <property name="text">
        <string>Station:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="textStation"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelParam">
       <property name="text">
        <string>Parameter:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboParam">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboType">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>TimeSeriesAdd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>TimeSeriesAdd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textStation</sender>
   <signal>editingFinished()</signal>
   <receiver>TimeSeriesAdd</receiver>
   <slot>onStationEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboParam</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TimeSeriesAdd</receiver>
   <slot>onParameterSelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStationEdited()</slot>
  <slot>onParameterSelected(int)</slot>
 </slots>
</ui>
